This is an ugly hax, but it's what upstream does, so i don't really care
to try and 'fix' it.

--- proxychains-2.1/proxychains/core.h
+++ proxychains-2.1/proxychains/core.h
@@ -68,4 +68,6 @@
 #ifndef __linux__
 #define __libc_connect _connect
+#else
+#define __libc_connect __connect
 #endif
 
